res/schema.xml

```diff
@@ -489,4 +489,12 @@ METADATA
       ALTER TABLE library ADD COLUMN color INTEGER;
     </sql>
   </revision>
+  <revision version="32" min_compatible="3">
```

Comment:
I suggest to use this opportunity for removing the (temporary) Rekordbox tables entirely from schema.xml like and instead switch to the same approach that we used for the Serato integration. This helps to avoid schema updates caused by optional components that don't affect the core functionality of Mixxx.

Any opinions?

Reply:
This should be possible, from what I can see the Serato integration just dynamically creates the temporary tables and drops them as needed. Would v32 of the DB schematic then just drop all previous permanent  Rekordbox tables?

Reply:
That was my idea to prevent further schema updates in the future.

Reply:
Actually after further thought v32 would be unnecessary as the dynamic tables would drop previous permanent Rekordbox tables created with v30. Should v30 then be updated to be a placeholder like v29?